WordPress/Sniff.php

```diff
@@ -252,6 +252,8 @@
 		'sanitize_email'             => true,
 		'sanitize_file_name'         => true,
 		'sanitize_html_class'        => true,
+		'sanitize_hex_color_no_hash' => true,
```

Comment:
On it! I should have just pulled the array out in phptester and sort it alphabetically XD